client/src/components/Visualization/charts/Interpolation.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { VictoryChart, VictoryLine, VictoryScatter } from 'victory';
+
+import PropTypes from 'prop-types';
+import { generateRandomHexColor } from '../../../utils/Utils';
+
+const MOCK_DATA = [
+  { x: 0, y: 0 },
+  { x: 1, y: 2 },
+  { x: 2, y: 1 },
+  { x: 3, y: 4 },
+  { x: 4, y: 3 },
+  { x: 5, y: 5 },
+];
+
+const cartesianInterpolations = [
+  'basis',
+  'bundle',
+  'cardinal',
+  'catmullRom',
+  'linear',
+  'monotoneX',
+  'monotoneY',
+  'natural',
+  'step',
+  'stepAfter',
+  'stepBefore',
+];
+
+const polarInterpolations = ['basis', 'cardinal', 'catmullRom', 'linear'];
+
+const InterpolationSelect = ({ currentValue, values, onChange }) => (
+  <select
+    onChange={onChange}
+    value={currentValue}
+    style={{ width: 'fit-content' }}
+  >
+    {values.map(value => (
+      <option value={value} key={value}>
+        {value}
+      </option>
+    ))}
+  </select>
+);
+
+InterpolationSelect.propTypes = {
+  currentValue: PropTypes.string,
+  values: PropTypes.array,
+  onChange: PropTypes.func,
+};
+
+const Interpolation = () => {
```

Comment:
Might be a good place for a docblock since "Interpolation" isn't the most intuitive function name